code/utils/scripts/lin_reg_plots.py

```diff
@@ -23,19 +24,18 @@
 smooth_data           = final_data + 'smooth/'
 hrf_data              = final_data + 'hrf/'
 
-
-#sys.path.append(os.path.join(os.path.dirname(__file__), "../functions/"))
```

Comment:
@janewliang 
Should we be doing the path append similar to line 27 or 28 for you (and linux users like Jarrod?)